tests/foreman/ui/test_package.py

```diff
@@ -12,26 +12,25 @@
 
 """
 from fauxfactory import gen_string
-from nailgun import entities
 import pytest
 
 from robottelo.config import settings
 from robottelo.constants import RPM_TO_UPLOAD, DataFile
 
 
 @pytest.fixture(scope='module')
-def module_org():
-    return entities.Organization().create()
+def module_org(module_target_sat):
+    return module_target_sat.api.Organization().create()
```

Comment:
same here

Reply:
please refer this comment for this https://github.com/SatelliteQE/robottelo/pull/13809#discussion_r1464795332